examples/dwarfdump.rs

```diff
@@ -247,12 +300,12 @@ where
     let debug_rnglists = load_section(file, endian);
     let rnglists = &gimli::RangeLists::new(debug_ranges, debug_rnglists)?;
 
+    let out = io::stdout();
     if flags.eh_frame {
-        dump_eh_frame(w, eh_frame)?;
+        dump_eh_frame(&mut BufWriter::new(out.lock()), eh_frame)?;
```

Comment:
These are a bit ugly, but I don't have a better idea. This relies on the drop to flush, right?

Reply:
Yes.